autoload/lsp/utils/text_edit.vim

```diff
@@ -44,42 +17,67 @@ function! lsp#utils#text_edit#apply_text_edits(uri, text_edits) abort
     " ((0, 4), (0, 5), "") - remove fifth character 'e'
     " ((0, 2), (0, 3), "") - remove third character 'c'
     let l:text_edits = sort(deepcopy(a:text_edits), '<SID>sort_text_edit_desc')
-    let l:i = 0
 
+    let l:path = lsp#utils#uri_to_path(a:uri)
+    let l:bufnr = bufnr(l:path, v:true)
+    if !bufloaded(l:path)
+        call bufload(l:path)
```

Comment:
Sorry, CI is failed on this line.
`bufload` seems to be a new api...

Reply:
I can't found the way to load buffer with no side-effect.
So I will close this PR later...

Reply:
@hrsh7th 
What about storing the current buffer in a variable, running `:edit`, and restoring the current buffer again?

Reply:
I trying to the solution, thanks. It works well.

https://github.com/prabirshrestha/vim-lsp/compare/master...hrsh7th:change-text-edit-implementation?expand=1